### DISCUSSION TITLE
Regarding Nuclei's yaml template, can all js paths in html be extracted through regular expressions and used as a dictionary to iterate a request, that is, each js path initiates a request?

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/nuclei#4202] [Q&A]
Regarding Nuclei's yaml template, can all js paths in html be extracted through regular expressions and used as a dictionary to iterate a request, that is, each js path initiates a request?

**Comment by MetzinAround** [comment]
Thanks for the question @hktalent. I'm not sure if they can be extracted through regex. @ehsandeep could you offer any insight?

**Reply by hktalent** [reply]
Thank you for your enthusiastic reply

I am sure that it is possible to extract and use a single variable, but the current scenario is to 
be able to store multiple extracted variables into variables and iterate. It is recommended to generate a new functional requirement.
That is to say, if there is 1 variable extracted by hand, it will be iterated only once, and if there are n variables, it will be iterated n times.
Scenario example:
1. Extract multiple possible tokens from the page, which may be tokens from different back-end servers, and then iterate one by one.
2. Extract all specific URLs in the page and iterate one by one

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
